src/atoms/Icon/Icon.web.js

```diff
@@ -1,3 +1,42 @@
-const Icon = () => null;
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { ThemeContext } from 'styled-components';
+import icons from '../../icons';
+import automation from '../../_helpers/automation-attributes';
+import { getColorKeys, getColor } from '../../_helpers/theme';
+
+const iconSize = {
+  xsmall: 12,
+  small: 16,
+  medium: 20,
+  large: 24,
+};
+
+const Icon = ({ size, name, fill, testID, ...rest }) => {
```

Comment:
Shouldn’t this be data-testid? testID is for native

Reply:
@kamleshchandnani 
the attribute returned from automation method will be `data-testid` but the param name is still testID.

Reply:
Sure. so in that case we don't need to expose this prop. This is required for react-native

for web we can just use `{...automation('ds-icon')}` and you can remove this from prop-types and everywhere